src/data/components/table/DataTable.tsx

```diff
@@ -323,38 +193,60 @@ function MuiVirtualizedTable({
   function setRef(ref: any) {
     ArrayRef = ref;
   }
+
   useEffect(() => {
     ArrayRef.recomputeRowHeights();
     ArrayRef.forceUpdate();
   }, [state, ArrayRef]);
 
   const widthList = [385, 222, 143, 390];
-  const consultantTableWidths = [385, 222, 117, 363, 53];
+  const consultantTableWidths = [385, 222, 143, 337, 53];
 
   const cellWidth = (index: number) =>
     columns.length === 5 ? consultantTableWidths[index] : widthList[index];
 
-  const cellRenderer: TableCellRenderer = ({
-    cellData,
-    columnIndex,
-    rowIndex,
+  const rowRenderer = (props: {
```

Comment:
Linter klager på manglende props validation

Reply:
En mulig løsning for å please ESLint her er å type `rowRenderer` som `TableRowRenderer` (importere fra `react-virtualized`), og så destrukturere props inne i parentesene:
```suggestion
  const rowRenderer: TableRowRenderer = ({
    className,
    index,
    key,
    rowData,
    style,
  }) => {
    // Trenger ikke destrukturere inne i funksjonen
    const id = rows[index].rowId;
```

(Husk i tilfelle å importere typen: `import { AutoSizer, Column, Table, TableRowRenderer } from 'react-virtualized';`)